src/renderer/dx/CustomTextRenderer.h

```diff
@@ -55,7 +55,7 @@ namespace Microsoft::Console::Render
         Outline
     };
 
-    constexpr const ULONG MinCursorHeightPercent = 25;
+    constexpr const ULONG MinCursorHeightPercent = 1;

```

Comment:
(oops, the comment didn't save)

Only thing I'm concerned about is some kind of side effect here.
1. @miniksa @DHowett Any ideas why this was set to 25 before? Will this horrifically explode in some way?
2. So if we set the value to something less than 25, the cursor height updates accordingly right? We're not clamping it to 25 anymore, so setting it to less than 25 actually get smaller?

Reply:
> I bet we specifically mention a minimum of 25 in the docs too. Mind making a PR over there and updating that too (then linking it here)?

Sure! But I could not found any mention of the minimum cursor height value in docs. Do you mean documentation in the `doc` folder or there is a separate repo for that?

The cursor becomes smaller on values less than 25, but if I set the value to less than 6, the cursor disappears. 

I probably found why the minimum value was set to 25:
https://github.com/microsoft/terminal/blob/ae550e0969595f062b15c2ff5cc33d4afe8ebc3f/src/buffer/out/cursor.h#L28-L30

Reply:
> Sure! But I could not found any mention of the minimum cursor height value in docs. Do you mean documentation in the doc folder or there is a separate repo for that?

Ah, yes. My bad. I'm thinking about [this section of the docs site](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/profile-appearance#cursor-height), in particular. That's generated by our [docs repo here](https://github.com/MicrosoftDocs/terminal).

> The cursor becomes smaller on values less than 25, but if I set the value to less than 6, the cursor disappears.
>
> I probably found why the minimum value was set to 25:

Hmm. That "large enough to be one pixel on a six pixel font" comment seems like a reason keep the minimum size to 25 imo. I think we should revert the changes to the schema and renderer, and change the minimum value on the slider to 25 then.

But I definitely want Michael or Dustin to check on this one to be safe (along with what you found in the `GdiEngine`).

Sorry for blocking on such an annoying little thing!

Reply:
So, if we revert the minimum value back to 25 shouldn't we also change the slider to number box? The slider looks prettier, but one with minimum value of 25 seems a little bit odd, in my opinion.

![cursorHeightSlider25](https://user-images.githubusercontent.com/39456018/110211557-e2422500-7eb8-11eb-822a-c995b2084d2d.gif)